src/webvr-polyfill.js

```diff
@@ -107,7 +107,7 @@ WebVRPolyfill.prototype.enablePolyfill = function() {
   navigator.getVRDisplays = this.getVRDisplays.bind(this);
 
   // Polyfill native VRDisplay.getFrameData
-  if (this.nativeWebVRAvailable && window.VRFrameData) {
+  if (this.nativeWebVRAvailable && this.isCardboardCompatible() && window.VRFrameData) {
```

Comment:
I understand better now. Should then the computation of `this.nativeWebVRAvailable` consider the value  of `this.isCardboardCompatible()`?